src/components/form.jsx

```diff
@@ -69,22 +69,24 @@ const Form = () => {
 
   return (
     <div className="bg-dark py-5 mb-12 md:mb-0 lg:py-12 px-[14px] dark:bg-ash">
-      <section className="block justify-center md:pb-16 md:flex items-center">
-        <div className="md:w-1/2 md:pr-20 md:text-left text-center">
-          <h2 className="text-purple font-bold text-xl lg:text-3xl dark:text-deeppurple">
+      <section className="block justify-center md:pb-16 md:flex md:flex-col lg:flex xl:flex lg:flex-row xl:flex-row items-center">
+        <div className="md:w-full lg:pr-20 lg:w-1/2 flex flex-col">
+          <h2 className="lg:text-left text-center text-purple font-bold text-xl md:text-2xl lg:text-3xl  dark:text-deeppurple">
             <span className="text-ash dark:text-dark">
               Start by entering a slang,
             </span>{" "}
             and our dictionary will spit out an abbreviation.{" "}
           </h2>
-          <p className="text-gray text-left lg:text-center text-sm mt-5">
+          <p className="text-gray text-leftlg:text-left md:text-center px-0 md:px-10 xmd:px-20 lg:px-0 xl:px-0 text-sm mt-5">
```

Comment:
I apologise for constantly reviewing, but you joined two classes here that I think should have a space in between.
```suggestion
          <p className="text-gray text-left lg:text-left md:text-center px-0 md:px-10 xmd:px-20 lg:px-0 xl:px-0 text-sm mt-5">
```